src/main/scala/notifications/kafka/Configs.scala

```diff
@@ -93,6 +96,7 @@ object Configs {
   }
 
   object Producer extends WeatherProducerConfig {
+    val delay = Duration(10, MINUTES)
```

Comment:
i literally meant you could import scala.concurrent.duration._ and use "10.minutes"

Reply:
it didn't show up in IntelliJ so I took the safe route